worker_server.py

```diff
@@ -13,6 +13,8 @@
 WORKER_HOST = "localhost"
 WORKER_PORT = 50052
 ADD_DELAY = False
+MAX_WORKERS = 2
```

Comment:
can we rename to max_thread_per_worker. MAX_WORKERS seems like a misnomer? Similar for other variables used for this purpose in the file

Reply:
Updated